src/lib/getData.ts

```diff
@@ -41,7 +41,7 @@ const getArrayValues = async (
 
   const arrayLength = await decodeKeyValue(
     'Number',
-    'uint128',
+    'uint128+',
```

Comment:
why is there a `+` sign here? This must be a typo

```suggestion
    'uint128',
```

Reply:
This means that we can read all the bytes to allow the old uint255 as well as the mistake uint16. When we see the plus then we read all the hex bytes and calculate how big the uintN should be. This is only used for the array length item